styles_src/default.scss

```diff
@@ -116,15 +130,19 @@ $image-margin: 7px;
         overflow: hidden;
         white-space: nowrap;
         text-overflow: ellipsis;
-        padding-left: 5px;
+        margin-top: 2px;
       }
 
       &-votes {
         margin-left: auto;
-        z-index: 1;
+        /*         z-index: 1;
```

Comment:
Can the z-index be removed here?

Reply:
yes, this is also a leftover of the pseudo transparency.